.github/workflows/platform-docker-trivy-scan.yml

```
@@ -101,7 +104,7 @@ jobs:
 
       - name: Scan ${{ matrix.image-name }}:main image
         id: trivy
-        uses: Flagsmith/actions/trivy-scan-image@v0.3.0
+        uses: Flagsmith/actions/trivy-scan-image@b88317a18754f7c6286d86eb50448c5225a366e0
```

```suggestion
        uses: Flagsmith/actions/trivy-scan-image@v0.4.0
```

after https://github.com/Flagsmith/actions/pull/5 is merged and released.